Traceback (most recent call last):
  File "/home/ck/Desktop/Projects/369/email_sender.py", line 39, in <module>
    with open("./data.json", 'r') as f:
         ~~~~^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: './data.json'
Email sent for Test Günü!
Failed to send email for Tıp Bayramı: (535, b'5.7.8 Username and Password not accepted. For more information, go to\n5.7.8  https://support.google.com/mail/?p=BadCredentials a640c23a62f3a-ac3147e9171sm225209066b.46 - gsmtp')
